ERROR - 2023-02-15 02:38:45 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'inventory', 3306, '', 0)
#1 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Handlers\DatabaseHandler.php(100): CodeIgniter\Database\BaseConnection->initialize()
#3 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_sessions')
#4 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Session.php(913): session_start()
#5 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Session.php(235): CodeIgniter\Session\Session->startSession()
#6 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\Services.php(665): CodeIgniter\Session\Session->start()
#7 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
#8 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#9 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#10 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
#11 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Common.php(932): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#12 C:\xampp\htdocs\inventory-management\app\Controllers\Auth.php(14): session()
#13 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(925): App\Controllers\Auth->index()
#14 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#15 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\inventory-management\public\index.php(67): CodeIgniter\CodeIgniter->run()
#17 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Handlers\DatabaseHandler.php(100): CodeIgniter\Database\BaseConnection->initialize()
#2 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_sessions')
#3 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Session.php(913): session_start()
#4 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Session\Session.php(235): CodeIgniter\Session\Session->startSession()
#5 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\Services.php(665): CodeIgniter\Session\Session->start()
#6 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
#7 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#8 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', NULL)
#9 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
#10 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\Common.php(932): CodeIgniter\Config\BaseService::__callStatic('session', Array)
#11 C:\xampp\htdocs\inventory-management\app\Controllers\Auth.php(14): session()
#12 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(925): App\Controllers\Auth->index()
#13 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#14 C:\xampp\htdocs\inventory-management\vendor\codeigniter4\framework\system\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\xampp\htdocs\inventory-management\public\index.php(67): CodeIgniter\CodeIgniter->run()
#16 {main}
CRITICAL - 2023-02-15 02:38:45 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Session\Handlers\DatabaseHandler.php(100): CodeIgniter\Database\BaseConnection->initialize()
 2 [internal function]: CodeIgniter\Session\Handlers\DatabaseHandler->open('ci_sessions', 'ci_sessions')
 3 SYSTEMPATH\Session\Session.php(913): session_start()
 4 SYSTEMPATH\Session\Session.php(235): CodeIgniter\Session\Session->startSession()
 5 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 6 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 7 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 8 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
 9 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
10 SYSTEMPATH\Common.php(932): CodeIgniter\Config\BaseService::__callStatic('session', [])
11 APPPATH\Controllers\Auth.php(14): session()
12 SYSTEMPATH\CodeIgniter.php(925): App\Controllers\Auth->index()
13 SYSTEMPATH\CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
14 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
15 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
